docs/inputs/input-group.md

```diff
@@ -42,3 +42,25 @@ As an example, the image below shows an input group with 5 different inputs (two
 ![img alt](/docs/inputs/input_group.png)
 
 </div>
+
+### Conditional Input Group
+
+| :trophy: This is a [premium](https://mecsimcalc.com/premium) features |
+| --------------------------------------------------------------------- |
+
+Click the pencil icon near the top right of an input group and the **Edit Input Group** popup will appear.  
+**Conditional Input Group** is similar to the [Conditional Input](https://docs.mecsimcalc.com//inputs/conditional-input) feature, but instead controls the visibility of the entire **Input Group**.
```

Comment:
I'd replace: "but instead controls" with smth like: "the only different is that condition input is applied to individual components whereas conditional input group is applied ot the entire input group that may contain multiple components. Although this is a bit longer

Reply:
You don't need to write the entire URL "https://docs.mecsimcalc.com//inputs/conditional-input", just "/inputs/conditional-input" should work iirc